### DISCUSSION TITLE
StatusBarBehaviour transparent background

### DISCUSSION THREAD

**Discussion by FM1973** [CommunityToolkit/Maui#1109] [Q&A]
Is there a way to set the background of the statusBarBehavior to transparent?
I tried #00FFFFFF but this doesn´t work.

**Comment by KieranMaclagan** [comment]
StatusBarColor="Transparent" works for us (at least on iOS)

**Comment by FM1973** [comment]
I can´t get it to run. Neither on IOS nor Android. I can set successfully any color, but transparent does not work.
Using latest toolkit version, latest maui (7.0.59) and current IOS as well as Android 11 to 13.

**Comment by FM1973** [comment]
Ok, seems to be because of the maui shell implementation:
[https://github.com/dotnet/maui/issues/9250](https://github.com/dotnet/maui/issues/9250)

**Comment by CliffAgius** [comment]
Closed and answered
